### PR TITLE
updated CC clusters

### PR DESCRIPTION
**Pull request by kowsisoundhar12**
Updated Following Cluster:
TC-CC-3.3
TC-CC-4.3
TC-CC-7.3
TC-CC-8.1

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_CC_3_3.yaml [hunk 1]
```diff
@@ -63,14 +63,25 @@ tests:
...
           command"
-      disabled: true
       command: "readAttribute"
       attribute: "current hue"
       response:
-          value: -1
+          constraints:
+              type: uint8
+              minValue: 0
+              maxValue: 254
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_CC_3_3.yaml:84]: But ... this is not checking that the value matches the value sent by the last command!

File: src/app/tests/suites/certification/Test_TC_CC_3_3.yaml [hunk 2]
```diff
@@ -87,14 +98,24 @@ tests:
...
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_CC_3_3.yaml:118]: Again, this is not doing what the test label says it should be doing.

File: src/app/tests/suites/certification/Test_TC_CC_4_3.yaml [hunk 3]
```diff
@@ -40,11 +40,13 @@ tests:
...
     - label: "Check Saturation attribute value matched before any change"
-      disabled: true
       command: "readAttribute"
       attribute: "current saturation"
       response:
-          value: 90
+          constraints:
+              type: uint8
+              minValue: 0
+              maxValue: 254
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_CC_4_3.yaml:49]: Again, does not match label.

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]
Please make sure the actual checks the test steps do match what they are claiming to do.

File: src/app/tests/suites/certification/Test_TC_CC_3_3.yaml (hunk 1 above)
Comment (kowsisoundhar12) [reply src/app/tests/suites/certification/Test_TC_CC_3_3.yaml:82]: Modified the label as per testplan

File: src/app/tests/suites/certification/Test_TC_CC_3_3.yaml (hunk 2 above)
Comment (kowsisoundhar12) [reply src/app/tests/suites/certification/Test_TC_CC_3_3.yaml:114]: Modified the label as per testplan

File: src/app/tests/suites/certification/Test_TC_CC_4_3.yaml (hunk 3 above)
Comment (kowsisoundhar12) [reply src/app/tests/suites/certification/Test_TC_CC_4_3.yaml:49]: Modified the label as per testplan

File: src/app/tests/suites/certification/Test_TC_CC_3_3.yaml [hunk 4]
```diff
@@ -39,14 +39,14 @@ tests:
       response:
           value: 1
 
-    - label:
-          "Check current hue attribute value matched the value sent by the last
-          command"
-      disabled: true
+    - label: "Reads CurrentHue attribute from DUT"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_CC_3_3.yaml:42]: What is the point of this step, if we don't do anything with the value we read?

Can you please point me to the test plan in question?

Comment (kowsisoundhar12) [reply src/app/tests/suites/certification/Test_TC_CC_3_3.yaml:42]: cc-3-3

@bzbarsky-apple above step is current hue attribute  value range check as per Testplan

**Review by bzbarsky-apple — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**